test/tests/google_vs_google_beta/expected/main.tf

```diff
@@ -3,7 +3,8 @@
 terraform {
   required_providers {
     google = {
-      source = "hashicorp/google"
+      source  = "hashicorp/google"
+      version = "4.84.0"
```

Comment:
need to lock the version... this is because the 'labels' was removed from beta state (now in main version).
therefore, need to lock to a version where this was beta.